ProcessOptimizer/model_systems/model_system.py

```diff
@@ -8,29 +11,29 @@ class ModelSystem:
 
     Parameters:
     * `score` [callable]:
-        Function for calculating the score of the system at a given point in 
-        the parameter space. It is expected to have the following signature:
-
-            def score(x, rng=np.random.default_rng(), noise_std=noise_value):
-                # * x is the point in the parameter space where the score will 
-                #   be evaluated. Note that x is in the original coordinates of
-                #   the parameter space, not normalized coordinates. 
-                # * rng is the random number generator used for adding noise 
-                #   to the system. The user can set a seed through this 
-                #   parameter. There must be a default value for this parameter. 
-                # * noise_std is the standard deviation of the noise. There
-                #   must be a default value for this parameter. 
-                ....
-                # The score of the system is returned
-                return score
+        Function for calculating the noiseless score of the system at a given point in 
+        the parameter space.
 
     * `space` [Space]:
         The parameter space in the form of a Space object. 
```

Comment:
"The parameter space in the form of a Space object". 
Because I'm relatively known in the package, I understand this as being an instantiated object of a space. As in `opt.space`. 
I believe that the intent is for the user to give a list of lists (or tubles) as is done during the initiation of the "opt"-object.

Reply:
Good point, I am going to make a space_factory() function in the space module, so we have the option of not caring whether we give the model system a space of the definitions of a space. This will also be useful in many other places, so it is worth having with space, and not in model_system.